nexusparser/tools/yaml2nxdl/yaml2nxdl_recursive_build.py

```diff
@@ -8,17 +8,26 @@
 import os, sys
```

Comment:
We have a common file header about license and authors that should be used.

Reply:
Is it the following?

#
# Copyright The NOMAD Authors.
#
# This file is part of NOMAD. See https://nomad-lab.eu for further info.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#

Reply:
ops.. the hashtag made it huge

Reply:
Is it the following text?


Copyright The NOMAD Authors.

 This file is part of NOMAD. See https://nomad-lab.eu for further info.

 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at

     http://www.apache.org/licenses/LICENSE-2.0

 Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.

Reply:
Yes. The project could also have an `AUTHORS` file. Something like `git shortlog -sne` can be used to get a list of all authors. This gives at least some credit.